sdk/containerregistry/azure-containerregistry/tests/test_container_repository_client.py

```diff
@@ -117,60 +155,139 @@ def test_list_tags_descending(self, containerregistry_baseurl):
 
         assert count > 0
 
-    @pytest.mark.live_test_only
+    @pytest.mark.live_test_only  # Recordings error, recieves more than 100 headers, not that many present
```

Comment:
are you waiting for this fix before merging?

Reply:
No, I'll make the fix in the next sprint.